test/request-response-samples/complexType_request/request.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
This is a malformed JSON file - unlike plain JavaScript objects, in JSON all Strings have to be double quoted -->
for example:
```json
{
  "ModelCRUDRequest": {
    "ModelCRUD": {
      "serviceType": "CreateInterest"
    }
  }
}